playbook/app/pb_kits/playbook/pb_table/table_colgroup.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Playbook
+  module PbTable
+    class TableColgroup < Playbook::KitBase
+      def classname
+        generate_classname("pb_table_colgroup_kit")
+      end
+    end
+  end
+end
```

Comment:
Whats the point of making this a kit...It doesn't do anything?

Reply:
has been removed